baselines/webworker.generated.d.ts

```diff
@@ -1484,6 +1486,25 @@ interface ImageBitmap {
     close(): void;
 }
 
+interface URL {
```

Comment:
just edit `inputfiles\knownWorkerInterfaces.json` to add `URL` and `URLSearcParams`.